app/views/shared/_admin_header.html.erb

```diff
@@ -0,0 +1,19 @@
+<html>
+<head>
+<style>
+  h1 {text-align: center;}
+</style>
+</head>
+<body>
+
+<h1>Little Esty Shop</h1>
+<h2>Admin Dashboard</h2>
+
+<%= link_to "Dashboard", admin_dashboard_index_path %>
```

Comment:
This is great.  I think we have a similar thing going on for the merchant user stories.